pkg/services/ngalert/sender/router.go

```diff
@@ -75,20 +75,19 @@ func NewAlertsRouter(multiOrgNotifier *notifier.MultiOrgAlertmanager, store stor
 // SyncAndApplyConfigFromDatabase looks for the admin configuration in the database
 // and adjusts the sender(s) and alert handling mechanism accordingly.
 func (d *AlertsRouter) SyncAndApplyConfigFromDatabase() error {
-	d.logger.Debug("start of admin configuration sync")
 	cfgs, err := d.adminConfigStore.GetAdminConfigurations()
 	if err != nil {
 		return err
 	}
 
-	d.logger.Debug("found admin configurations", "count", len(cfgs))
+	d.logger.Info("Attempting to sync admin configs", "count", len(cfgs))
 
 	orgsFound := make(map[int64]struct{}, len(cfgs))
 	d.adminConfigMtx.Lock()
 	for _, cfg := range cfgs {
 		_, isDisabledOrg := d.disabledOrgs[cfg.OrgID]
 		if isDisabledOrg {
-			d.logger.Debug("skipping starting sender for disabled org", "org", cfg.OrgID)
```

Comment:
I think it should be a debug or even remove it at all 
because 
1. this is not something user can fix (for example when they run Grafana with all alerting and then disabled it for specific orgs.
2. This is the normal use case.

I think we should query for only enabled orgs, i.e. move disabled orgs to the store layer (same as in rule store)

Reply:
Ah, context makes more sense now. I removed entirely